frappe/desk/query_report.py

```diff
@@ -194,6 +197,9 @@ def run(
 
 	result = None
 
+	if (is_default_filters == True or is_default_filters == "true") and report.custom_filters:
```

Comment:
```suggestion
	if frappe.parse_json(is_default_filters) and report.custom_filters:
```

Reply:
Thanks

Reply:
`sbool`? - instead of `json.loads`-ing it

Reply:
Thanks, didn't know about that